Gek62/hometask-4/my_array_rotate_benchmark.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'benchmark'
+require './my_array_rotate'
```

Comment:
If I run this file from a different directory, I get the error:
```
cannot load such file -- ./my_array_rotate (LoadError)
```

Using `require_relateve` should fix it